cmake/Dependencies.cmake

```diff
@@ -537,7 +537,7 @@ if(BUILD_CAFFE2 OR BUILD_ATEN)
     hip_include_directories(${Caffe2_HIP_INCLUDES})
 
     set(Caffe2_HIP_DEPENDENCY_LIBS
-      ${rocrand_LIBRARIES} ${hiprand_LIBRARIES} ${PYTORCH_HIP_HCC_LIBRARIES} ${PYTORCH_MIOPEN_LIBRARIES} ${hipblas_LIBRARIES})
+      ${rocrand_LIBRARIES} ${hiprand_LIBRARIES} ${PYTORCH_HIP_HCC_LIBRARIES} ${PYTORCH_MIOPEN_LIBRARIES})
```

Comment:
We don't need a `${rocblas_LIBRARIES}` here?

Reply:
Rest looks good afaict

Reply:
No, we don't, it adds rocBLAS a few lines below anyways (hipBLAS always relied on rocBLAS in the backend).